tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -994,6 +1014,59 @@ public function userGetsAllTheMembersOfGroupUsingTheGraphApi($user, $group): voi
 		$this->featureContext->setResponse($this->listGroupMembers($group, $user));
 	}
 
+	/**
+	 * @When user :user retrieves all groups along with their members using Graph API
+	 * @When user :user gets all the members information of group :group using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $group
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userRetrievesMembersOfAllGroupsUsingTheGraphAPI(string $user, string $group = ''): void {
+		$this->featureContext->setResponse($this->listAllGroupsAlongWithAllMemberInformation($user, $group));
+	}
+
+	/**
+	 * @Then the group :group should have the following member information
+	 *
+	 * @param string $group
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function theGroupShouldHaveTheFollowingMemberInformation(string $group, TableNode $table): void {
+		$response = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse());
+		$rows = $table->getHash();
+		$nextMemberInformation = 0;
+		if (isset($response['value'])) {
+			$response = $response['value'];
+			$groupFoundInResponse = false;
+			foreach ($response as $value) {
```

Comment:
if we don't use $response = $response['value']; in multiple place then we can directly use like this?
```suggestion
		if (isset($response['value'])) {
			$groupFoundInResponse = false;
			foreach ($response['value'] as $value) {
```

Reply:
the reponse basically is either `$response` or `$response['value']`. if the reponse does not contains `['value']` then it only checks for `$response` part.